documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,183 @@
+
+## (Optional) Install the `cx-server` Toolkit for Jenkins
+
+`cx-server`is a lifecycle management toolkit that provides Docker images with a preconfigured Jenkins and a Nexus-based cache to facilitate the configuration and usage of Jenkins.
+
+To use the toolkit, get the `cx-server` script and its configuration file `server.cfg` by using the following command:
+
+```sh
+docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
+```
+
+When the files are downloaded into the current directory, launch the Jenkins server by using the following command:
+
+
+```sh
+./cx-server start
+```
+
+For more information on the Jenkins lifecycle management and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server](https://github.com/SAP/devops-docker-images/blob/master/docs/operations/cx-server-operations-guide.md). 
+
+
+## Create Your First Pipeline
+
+For the beginning, we recommend using an SAP sample application. The repository [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] contains a simple `nodejs` application that can be enriched with a pipeline built with MTA and deployed into the Cloud Foundry environment.
+
+1. Fork the [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] repository into your GitHub organization.
+
+1. Select the `1_REST_persist_in_Memory` branch of your [cloud-cf-helloworld-nodejs] fork and in it, create a new file with the name `Jenkinsfile`.
+
+1. To synchronize the repository, enter the following code lines into your `Jenkinsfile`: 
+   ```
+   @Library('piper-lib-os') _
+    node() {
+    stage('prepare') {
+      checkout scm
+      setupCommonPipelineEnvironment script:this
+      }
+    }
+   ```
+   For more information about Jenkinsfiles and pipelines, see [Using a Jenkinsfile][jenkins-io-jenkinsfile].
+   
+1. To set up a Jenkins job for your repository, open the Jenkins UI under `http://<jenkins-server-address>:<http-port>` and choose **New Item**. Per default, the `cx-server` starts Jenkins on HTTP port `80`. For more information, see the [Jenkins User Documentation][jenkins-io-documentation].
+   <p align="center">
+   ![Clicke New Item](../images/JenkinsHomeMenu-1.png "Jenkins Home Menu")
+   </p> 
+   
+1. Provide a name for your new item (for example, *My First Pipeline*) and select **Pipeline**.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   </p>  
+
+1. For **Definition** in the **Pipeline** options, choose **Pipeline script from SCM**. 
+
+1. For **SCM**, choose **Git**.
+
+1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
```

Comment:
It should/could be just one option of a list of supported. I guess there are restrictions, aren't there?

next step i guess

Reply:
of course we can handle this with a "next step" approach. But experience teaches that it will probably stay like that forever. IMO the decission to which level we are agnostic/neutral to some git incarnations is important. I see the tension between being platform agnostic and having a step-by-step guide which must be simple and easy to follow ...